recipes/fleur/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "fleur" %}
+{% set version = "6.0" %}
+{% set version_long = "MaX-R" + version %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://iffgit.fz-juelich.de/{{ name }}/{{ name }}/-/archive/{{ version_long }}/{{ name }}-{{ version_long }}.tar.gz
+  sha256: b022793360c198e18f269541152a04189990c0bf8326939f0f24bb8ca2adf757
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - openmpi-mpifort
+    - cmake
+    - make
+    - git
+    - doxygen
+  host:
+    - libblas
+    - liblapack
+    - libgomp
+    - openmpi
+    - scalapack
+    - libxml2
+    - fftw
+    - spfft
```

Comment:
Does your package need spfft at runtime? The build logs say that this library was [not found during the build process.](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=457112&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1426)

Then we get [overdependency warnings from conda-build.](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=457112&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2483)

Reply:
Spfft is an optional dependency that is probably not configured correctly in the recipe.
@ltalirz I think we can remove it for now since fftw is already included correctly

Reply:
thanks for the feedback guys; spfft removed